resource/css/skosmos.css

```diff
@@ -204,6 +205,7 @@ body {
   }
 
   #search-wrapper {
+    display: flex;
```

Comment:
This line seems to be overwritten by something else. If flex is needed for the element, `!important` should be added.

Reply:
Damn, that was the only thing that remained when I was trying out a different layout solution. It was needed then, but it seems it is not needed in the current layout.